src/validateOptions.js

```diff
@@ -24,7 +28,18 @@ const validateOptions = (schema, options, name) => {
   }
 
   if (!ajv.validate(schema, options)) {
-    throw new ValidationError(ajv.errors, name);
+    try {
+      throw new ValidationError(ajv.errors, name);
+    } catch (err) {
+      console.error(chalk.bold.red(`\n${err.message}\n`));
```

Comment:
This could be ```console.error(chalk.bold.red(`\n${err.toString()}\n`))``` instead as a nitpick :bird:, in case the `err.name` should be included